adapters/conversant/conversant.go

```diff
@@ -130,6 +139,18 @@ func parseCnvrParams(imp *openrtb2.Imp, cnvrExt openrtb_ext.ExtImpConversant) {
 			imp.Video.MaxDuration = *cnvrExt.MaxDuration
 		}
 	}
+	if imp.BidFloor > 0 && imp.BidFloorCur != "" && strings.ToUpper(imp.BidFloorCur) != "USD" {
+		floor, err := reqInfo.ConvertCurrency(imp.BidFloor, imp.BidFloorCur, "USD")
+		if err != nil {
+			return []error{&errortypes.BadInput{
+				Message: fmt.Sprintf("Unable to convert provided bid floor currency from %s to USD", imp.BidFloorCur),
+			}}
+		} else if floor > 0 {
+			imp.BidFloorCur = "USD"
+			imp.BidFloor = floor
+		}
+	}
```

Comment:
Please add the json test case for this change.

Reply:
I have added this to one of the test cases